### DISCUSSION TITLE
how to support "closed testing" for the track

### DISCUSSION THREAD

**Discussion by sk92129** [Triple-T/gradle-play-publisher#1021] [General]
Hi,
How do I use this tool to publish to "Closed Testing" instead of "Internal Testing"?

It works for me when I use the default or :
play {
    defaultToAppBundles = true
    track = "internal" // 'alpha', 'beta', 'rollout' or 'production'
    // it isnt clear what the format should be for this serviceAccess.json and how to protect content
    serviceAccountCredentials = file("serviceAccess.json")
    //resolutionStrategy.set(ResolutionStrategy.AUTO)
}
But if I changed the track to "closed", "closed testing", i get an error.

     {
       "code": 400,
       "errors": [
         {
           "domain": "global",
           "message": "Track not found: closed",
           "reason": "badRequest"
         }
       ],
       "message": "Track not found: closed",
       "status": "INVALID_ARGUMENT"
     }

**Comment by SUPERCILEX** [comment]
https://github.com/Triple-T/gradle-play-publisher#uploading-an-internal-sharing-artifact

**Reply by sk92129** [reply]
I see anything here mention about closed testing.  But i found another approach by uploading to internal and promoting manually to closed testing

**Reply by SUPERCILEX** [reply]
Oops, thought you meant "Internal app sharing."

`alpha` should go to the play console "Closed testing" section.
